src/snippets/langsmith/framework-agnostic.mdx

```diff
@@ -0,0 +1 @@
+LangSmith Deployment supports deploying a [LangGraph](/oss/python/langgraph/overview) _graph_. However, the implementation of a _node_ of a graph can contain arbitrary code. This means any framework can be implemented within a node and deployed on LangSmith Deployment. This lets you keep your core application logic outside LangGraph while still using LangSmith for [deployment](/langsmith/deployments), scaling, and [observability](/langsmith/observability). For more details, refer to [Use any framework with LangSmith Deployment](/langsmith/application-structure#use-any-framework-with-langsmith-deployment).
```

Comment:
Small update to this (I think I suggested the original copy): `This lets you keep your core application logic outside LangGraph` --> `This lets you implement your core application logic without using additional LangGraph OSS APIs`

Not sure if more or less confusing. I just want to be a direct and literal as possible. Words like `outside` sound too physical.